testing/features/support/components/notice_banner/notice_banner.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module NoticeBanner
+  class Example < ::Base
+    set_url "/iframe.html?id=components-notice-banner--example&args=&viewMode=story"
+
+    element :notice_banner_title, ".cads-notice-banner__title"
```

Comment:
Would be good to remove the styling class and use something like:
`element :notice_banner_title, "span", { text: "Notice banner title" }` instead

Reply:
Done!